TerminalDocs/json-fragment-extensions.md

```diff
@@ -59,31 +60,52 @@ The JSON file should be split up into 2 lists, one for profiles and one for sche
 }
 ```
 
-The first item in the `"profiles"` list updates an existing profile, identifying the profile it wishes to update via the GUID provided to the `"updates"` field (details on how to obtain the GUID are in the next section). The second item in that list creates a new profile called "Cool Profile".
+The first item in the `"profiles"` list updates an existing profile, identifying the profile it wishes to update via the GUID provided to the `"updates"` field (details on how to obtain the GUID are in the next section). The second item in that list creates a new profile called "Cool Profile" with a predetermined GUID. Providing such a GUID allows users to customize a profile regardless of whether its name or location changes.
 
 In the `"schemes"` list, a new color scheme called "Postmodern Tango Light" is defined, and can be subsequently referenced by the user in their settings file or in this JSON file itself (notice that "Cool Profile" uses this newly defined color scheme).
 
 Of course, if the developer only wishes to add/modify profiles without adding color schemes (and vice-versa), only the relevant list needs to be present and the other list can be omitted.
 
-## How to determine the GUID of an existing profile 
+## Profile GUIDs
 
-The only profiles that can be modified through fragments are the default profiles, Command Prompt and PowerShell, as well as [dynamic profiles](./dynamic-profiles.md). To determine the GUID of the profile to be updated, use a Version 5 UUID generator with the following namespace GUID and name:
+As previously stated profile GUIDs are a way to reference profiles and let users update and extend them without worrying about location or name changes. The only profiles that can be modified through fragments are the default profiles, Command Prompt and PowerShell, as well as [dynamic profiles](./dynamic-profiles.md). Providing a GUID is optional, however, strongly encouraged.
 
-- The namespace GUID: `{2BDE4A90-D05F-401C-9492-E40884EAD1D8}` 
-- The name of the profile to be updated 
+The GUIDs are generated using a Version 5 UUID generator which supports BOM-less UTF-16LE encoding. The namespace GUID for Windows Terminal is `{2BDE4A90-D05F-401C-9492-E40884EAD1D8}`.
```

Comment:
```suggestion
The GUIDs are generated using a Version 5 UUID generator which supports BOM-less UTF-16LE encoding. The namespace GUID for Windows Terminal is `{f65ddb7e-706b-4499-8a50-40313caf510a}`.
```

Based on @DHowett's [comment](https://github.com/microsoft/terminal/issues/10374#issuecomment-857830194), I believe this is the correct namespace GUID.